build.gradle

```diff
@@ -51,7 +51,7 @@ dependencies {
     compileOnly group: "io.kestra", name: "script", version: kestraVersion
 
     // kestra API client
-    implementation "io.kestra:kestra-api-client:1.0.1"
+    implementation "io.kestra:kestra-api-client:1.0.2"
```

Comment:
Strange I thought we were in a far bigger version for the SDK?

Reply:
there was some rework around versioning recently, this is indeed the correct version https://github.com/kestra-io/client-sdk/releases/tag/v1.0.2-java